### PR TITLE
Added examples in keras, caret, and nnet

### PR DESCRIPTION
**Pull request by dsblank**
Added some examples for beta users to be able to have some "getting started" demos.

### CODE REVIEW

File: inst/train-examples/caret-example.R [hunk 1]
```diff
@@ -45,15 +45,22 @@ dataset <- dataset[validation_index,]
 x <- dataset[,1:4]
 y <- dataset[,5]
 # scatterplot matrix
-# Scatterplot shows overlap of green (species "virginica") 
+# Scatterplot shows overlap of green (species "virginica")
```
Comment (Nimrod007) [inline inst/train-examples/caret-example.R:48]: add comments on L18 create_experiment explaining users how to setup the config and link to docs

File: inst/train-examples/keras-example.R [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+# library(tensorflow)
+# install_keras()
+
+library(cometr)
+library(tidyr)
+library(ggplot2)
+library(keras)
+library(reticulate)
+
+exp <- create_experiment(
```
Comment (Nimrod007) [inline inst/train-examples/keras-example.R:16]: add comments  explaining users how to setup the config and link to docs

File: inst/train-examples/nnet-example.R [hunk 3]
```diff
@@ -0,0 +1,103 @@
+# Based on:
+# https://www.rdocumentation.org/packages/nnet/versions/7.3-13/topics/nnet
+
+library(cometr)
+library(nnet)
+library(stringr)
+
+exp <- create_experiment(
```
Comment (Nimrod007) [inline inst/train-examples/nnet-example.R:8]: add comments  explaining users how to setup the config and link to docs

File: inst/train-examples/caret-example.R (hunk 1 above)
Comment (dsblank) [reply inst/train-examples/caret-example.R:57]: Done!

File: inst/train-examples/keras-example.R (hunk 2 above)
Comment (dsblank) [reply inst/train-examples/keras-example.R:25]: Done!

File: inst/train-examples/nnet-example.R (hunk 3 above)
Comment (dsblank) [reply inst/train-examples/nnet-example.R:18]: Done!

**Pull request merged**